Sources/PointFree/Account/Account.swift

```diff
@@ -103,7 +103,7 @@ private let accountView = View<AccountData> { data in
     gridColumn(sizes: [.mobile: 12, .desktop: 8], [style(margin(leftRight: .auto))], [
       div([`class`([Class.padding([.mobile: [.all: 3], .desktop: [.all: 4]])])],
           titleRowView.view(unit)
-            <> profileRowView.view((data.currentUser, data.emailSettings))
+            <> profileRowView.view(data)
```

Comment:
this view grew to need more and more data from the `ProfileData`, so just passing the whole thing along now.